### PR TITLE
Improvement: Request Response Clinet handle errors

### PR DESCRIPTION
**Pull request by lciolecki**

### CODE REVIEW

File: engine/lite/request-response/runtime/src/main/scala/pl/touk/nussknacker/engine/requestresponse/management/RequestResponseProcessClient.scala [hunk 1]
```diff
@@ -113,5 +118,21 @@ class HttpRequestResponseClient(managementUrl: String)(implicit backend: SttpBac
 
   override def close(): Unit = Await.result(backend.close(), Duration(10, TimeUnit.SECONDS))
 
+  private def handleUnitResponse(action: String)(response: Response[Either[String, String]]): Future[Unit] = (response.code, response.body) match {
```
Comment (mproch) [inline engine/lite/request-response/runtime/src/main/scala/pl/touk/nussknacker/engine/requestresponse/management/RequestResponseProcessClient.scala:121]: Can you move it to httputils, extracting also from flink client?

**Review by dswiecki — COMMENTED** [review]
Please add changelog entry

**Comment by lciolecki** [comment]
> Please add changelog entry

Earlier.. @arkadius said that we shouldn't add everything to changelog.. IMO this change is not so important..

File: engine/lite/request-response/runtime/src/main/scala/pl/touk/nussknacker/engine/requestresponse/management/RequestResponseProcessClient.scala (hunk 1 above)
Comment (lciolecki) [reply engine/lite/request-response/runtime/src/main/scala/pl/touk/nussknacker/engine/requestresponse/management/RequestResponseProcessClient.scala]: Done, can you look right now?

**Review by mproch — APPROVED** [review]

**Pull request merged**